statefun-flink/statefun-flink-core/src/main/java/org/apache/flink/statefun/flink/core/jsonmodule/Pointers.java

```diff
@@ -48,6 +48,8 @@ private Pointers() {}
     public static final JsonPointer FUNCTION_STATES = JsonPointer.compile("/function/spec/states");
     public static final JsonPointer FUNCTION_TIMEOUT =
         JsonPointer.compile("/function/spec/timeout");
+    public static final JsonPointer FUNCTION_MAX_HTTP_BATCH_SIZE =
+        JsonPointer.compile("/function/spec/maxBatchSize");
```

Comment:
I'm wondering should we rather call this `maxNumBatchRequests` or something along those lines.
Just to clearly avoid confusion that its the actual batch data size of the HTTP request, and also to leave room for us to add such a data-size-based threshold in the future.

My reasoning is:
Just seeing it from the user perspective, I would personally use this by doing: `desiredMaxBatchDataSize / averageExpectedMessageSize` to control the size of the HTTP requests.
I can see a reason in the future to add such a size-based threshold.
This _might_ in some cases be more intuitive to use than a request-count-based threshold.

Reply:
But of course, that is something we can add in the future and not now.
Just suggesting a config key rename here, to leave as room for doing that in the future.

Reply:
Sounds good, lets change to that!
If you'd like you can change it while merging, or I'll push a hot fix to this PR

Reply:
I'll change while merging 👌